From 8ae2c803c548d34b827a37e80624ff42fbda3af8 Mon Sep 17 00:00:00 2001
From: Johnathan Mantey <johnathanx.mantey@intel.com>
Date: Tue, 7 Jun 2022 09:24:36 -0700
Subject: [PATCH] Remove read-only flag from NTPServers

The NTPServers API no longer permits static assignment of
NTPServers. This is a result of adding the read-only flag to the
NTPServers D-Bus element.

The API needs to allow submitters to assign NTPServers. This entry
needs to allow read/write operations.

Tested:
After removing the read-only flag a Restclient PATCH request to the
NTPServers succeeds. Inspection of the systemd.network configuration
file confirms the values are propagated to the phosphor-network
receiver.

Change-Id: Ia2944738fe2bb9c5312207b1df21d8be9ddc5af2
Signed-off-by: Johnathan Mantey <johnathanx.mantey@intel.com>

[NOTICE]
This patch is cherry-pick from commit 998c7bed.
Please remove it if it's already exists
---
 .../openbmc_project/Network/EthernetInterface.interface.yaml    | 2 --
 1 file changed, 2 deletions(-)

diff --git a/yaml/xyz/openbmc_project/Network/EthernetInterface.interface.yaml b/yaml/xyz/openbmc_project/Network/EthernetInterface.interface.yaml
index 155faa7..9e29d32 100644
--- a/yaml/xyz/openbmc_project/Network/EthernetInterface.interface.yaml
+++ b/yaml/xyz/openbmc_project/Network/EthernetInterface.interface.yaml
@@ -55,8 +55,6 @@ properties:
           - xyz.openbmc_project.Common.Error.InvalidArgument
     - name: NTPServers
       type: array[string]
-      flags:
-          - readonly
       description: >
           NTP servers that are currently in use on this interface.
           This includes both DHCP and static NTP server addresses.
-- 
2.33.0

